client-service/src/main/java/com/registry/clientservice/exception/ClientNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.registry.clientservice.exception;
+
+public class ClientNotFoundException extends RuntimeException{
```

Comment:
Same goes for this. I see that you have an InvoiceNotFoundException class in this PR as well. Look into combining them into a common NotFoundException.